modules/cnf-installing-the-n3000-operator.adoc

```diff
@@ -0,0 +1,162 @@
+
+The OpenNESS Operator for Intel FPGA PAC N3000 orchestrates and manages the resources or devices exposed by the Intel FPGA PAC N3000 card within the {product-title} cluster.
+
+For vRAN use cases, the OpenNESS Operator for Intel FPGA PAC N3000 is used with the OpenNESS Operator for Wireless FEC Accelerators.
+
+As a cluster administrator, you can install the OpenNESS Operator for Intel® FPGA PAC N3000 by using the {product-title} CLI or the web console.
+
+[id="install-operator-cli_{context}"]
+== Installing the Operator by using the CLI
+
+As a cluster administrator, you can install the Operator by using the CLI.
+
+.Prerequisites
+
+* A cluster installed on bare-metal hardware.
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a namespace for the N3000 Operator by completing the following actions:
+
+.. Define the `vran-acceleration-operators` namespace by creating a file named `n3000-namespace.yaml` file as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: vran-acceleration-operators
+    labels:
+        openshift.io/cluster-monitoring: "true"
+----
+
+.. Create the namespace by running the following command:
++
+[source,terminal]
+----
+$ oc create -f n3000-namespace.yaml
+----
+
+. Install the N3000 Operator in the namespace you created in the previous step:
+
+.. Create the following `OperatorGroup` CR and save the YAML in the `n3000-operatorgroup.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+    name: n3000-operators
+    namespace: vran-acceleration-operators
+spec:
+    targetNamespaces:
+    - vran-acceleration-operators
+----
+
+.. Create the `OperatorGroup` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f n3000-operatorgroup.yaml
+----
+
+
+.. Run the following command to get the `channel` value required for the next step.
++
+[source,terminal]
+----
+$ oc get packagemanifest n3000 -n openshift-marketplace -o jsonpath='{.status.defaultChannel}'
+----
++
+.Example output
+[source,terminal]
+----
+stable
+----
+
+.. Create the following `Subscription` CR and save the YAML in the `n3000-sub.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: n3000-subscription
+    namespace: vran-acceleration-operators
+spec:
+    channel: "<channel>"        <1>
+    name: n3000
+    source: certified-operators <2>
+    sourceNamespace: openshift-marketplace
+----
+<1> Specify the value for channel from the value obtained in the previous step for the `.status.defaultChannel` parameter.
+<2> You must specify the `certified-operators` value.
+
+.. Create the `Subscription` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f n3000-sub.yaml
+----
+
+. Verify that the OpenNESS Operator for Intel® FPGA PAC N3000 Operator is installed:
```

Comment:
I apologize for failing to make an identical request that I made for the "fec-operator" file.  Please...

* Add a `.Verification` block title.
* Change the "." to "*" because it is a single-step procedure.
* Change the instructional text to "Verify the Operator is installed:" so it matches the fec-operator file.